AshleyLab/motrpac-frontend
# MOTR-120 MOTR-136 Updated Open Graph preview image resize

**Key change**
- Adjusted the logo image to 1200x628 pixels to optimize a great thumbnail preview 

**How to test using localtunnel:**
- Install Localtunnel : `$ npm install -g localtunnel`
- Use the command line interface to request a tunnel to your local server: `$ lt -p 3000`
- You will receive a url, for example https://bitter-goose-76.localtunnel.me/, that you can type it on Facebook to see the preview image.

**Test Result:** 
- I tested by hosting it using localtunnel to check the updated preview image. The screenshot below is on Facebook with the update image logo.
![Screen Shot 2019-07-18 at 3 07 29 PM](https://user-images.githubusercontent.com/14895583/61496502-2f365800-a971-11e9-9059-03f1499f2441.png)

**Note**: 
- In prod environment, we will need to test absolute URLs for images since the relative path on og:image meta tag will result in broken preview image.
- On Slack, the Open Graph preview image size looks small as below due to their SlakcBot (https://api.slack.com/robots)
![Screen Shot 2019-07-18 at 3 35 38 PM](https://user-images.githubusercontent.com/14895583/61496682-df0bc580-a971-11e9-9f21-f81b1fe74acd.png)
